modules/op-release-notes-1-12.adoc

```diff
@@ -0,0 +1,192 @@
+
+* With this update, the following parameters in the `TektonResult` CR are no longer supported:
+** `db_user`
+** `db_password`
+** `s3_bucket_name`
+** `s3_endpoint`
+** `s3_hostname_immutable`
+** `s3_region`
+** `s3_access_key_id`
+** `s3_secret_access_key`
+** `s3_multi_part_size`
++
+You must provide these parameters using secrets. You must delete and re-create the `TektonResult` CR after upgrading to {pipelines-shortname} 1.12 to provide these parameters.
+
+* With this update, the `tkn pac bootstrap` command supports the `--github-hostname` flag. The `--github-api-url` flag is deprecated.
+
+[id="known-issues-1-12_{context}"]
+== Known issues
+
+* If limit ranges are configured for a namespace, but pod ephemeral storage is not configured in the limit ranges, pods can go into an error stage with the message `Pod ephemeral local storage usage exceeds the total limit of containers 0`.
+
+* If you want to make changes to the configuration in the `TektonResult` CR, you must delete the existing `TektonResult` CR and then create a new one. If you change an existing `TektonResult` CR, the changes are not applied to the existing deployment of {tekton-results}. For example, if you change the connection from an internal database server to an external one or vice versa, the API remains connected to the old database.
+
+[id="fixed-issues-1-12_{context}"]
+== Fixed issues
+
+* Before this update, {pac} ran pipeline runs based only on branch base names, and could  incorrectly trigger pipeline runs with the same base name but different branch name. With this update, {pac} does checks both the base name and the exact branch name of a pipeline run.
+
+* Before this update, an incoming webhook event could trigger multiple pipeline runs configured for other events. With this update, an incoming webhook event triggers only a pipeline run configured for the webhook event.
+
+* With this update, the `pac-gitauth` secrets are now explicitly deleted when cleaning up a pipeline run, in case the `ownerRef` on the pipeline run gets removed.
+
+* Before this update, when a task in a pipeline run failed with a reason message, the entire pipeline run failed with a `PipelineValidationFailed` reason. With this update, the pipeline run fails with the same reason message as the task that failed.
+
+*  Before this update, the `disable-ha` flag was not correctly passed to the Pipelines controller, and the high availability (HA) functionality was never enabled. With this update, the HA functionality is enabled if you set the `disable-ha` flag to `false`.
+
+* Before this update, the `skopeo-copy` cluster task would fail when attempting to copy images mentioned in  ConfigMap data. With this update, the `skopeo-copy` cluster task completes properly.
```

Comment:
Use `ConfigMap` only when referring to an API object. If not, use the generic form config map: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references